docs/usage/prometheus-metrics.md

```diff
@@ -27,3 +27,16 @@ We provide a {download}`sample Grafana dashboard </_downloads/grafana/liqonetwor
 As presented in the screenshot below, it includes an overview section presenting the overall cross-cluster throughput, followed by detailed per-peering throughput and latency information.
 
 ![Grafana Network Dashboard](/_static/images/usage/prometheus-metrics/network-dashboard.png)
+
+## Virtual kubelet metrics
+
+These metrics are available for each peered remote cluster, providing statistics about the reflected resources:
+
+- **liqo_virtual_kubelet_reflection_item_counter**: the number of reflected resources successfully processed (e.g., Pod, ConfigMap, Secret, Service, ServiceAccount, EndpointSlice, Ingress and PersistentVolumeClaim).
```

Comment:
```suggestion
- **liqo_virtual_kubelet_reflection_item_counter**: the number of resources that are currently successfully reflected (e.g., Pod, ConfigMap, Secret, Service, ServiceAccount, EndpointSlice, Ingress and PersistentVolumeClaim). This number can increase/decrease over time, and it may reach zero when two peered clusters have no reflected resources.
```